src/test/java/org/opentripplanner/graph_builder/module/geometry/CalculateWorldEnvelopeModuleTest.java

```diff
@@ -0,0 +1,48 @@
+package org.opentripplanner.graph_builder.module.geometry;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.opentripplanner.transit.model._data.TransitModelForTest.stop;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.street.model.vertex.Vertex;
+import org.opentripplanner.transit.model.site.RegularStop;
+
+class CalculateWorldEnvelopeModuleTest {
+
+  public static final String FEED_ID = "F";
+
```

Comment:
unused?

Reply:
Correct, try to use one of the existing Vertex classes but gave up, and created my own...